bgpd/bgp_route.h

```diff
@@ -316,15 +324,16 @@ struct bgp_path_info {
 
 	unsigned short instance;
 
+#define BGP_MPLSVPN_LABEL_NH (1 << 1)
+	uint8_t mplsvpn_usage;
```

Comment:
This leaves a hole of 7 bytes

pahole /usr/lib/frr/bgpd | grep -A 100 -m1  '^struct bgp_path_info'
```
struct bgp_path_info {
        struct bgp_path_info *     next;                 /*     0     8 */
        struct bgp_path_info *     prev;                 /*     8     8 */
        struct {
                struct bgp_path_info * le_next;          /*    16     8 */
                struct bgp_path_info * * le_prev;        /*    24     8 */
        } nh_thread;                                     /*    16    16 */
        struct bgp_node *          net;                  /*    32     8 */
        struct bgp_nexthop_cache * nexthop;              /*    40     8 */
        struct peer *              peer;                 /*    48     8 */
        struct attr *              attr;                 /*    56     8 */
        /* --- cacheline 1 boundary (64 bytes) --- */
        struct bgp_path_info_extra * extra;              /*    64     8 */
        struct bgp_path_info_mpath * mpath;              /*    72     8 */
        time_t                     uptime;               /*    80     8 */
        int                        lock;                 /*    88     4 */
        uint32_t                   flags;                /*    92     4 */
        uint8_t                    type;                 /*    96     1 */
        uint8_t                    sub_type;             /*    97     1 */
        short unsigned int         instance;             /*    98     2 */
        uint8_t                    mplsvpn_usage;        /*   100     1 */

        /* XXX 3 bytes hole, try to pack */

        uint32_t                   addpath_rx_id;        /*   104     4 */
        struct bgp_addpath_info_data tx_addpath;         /*   108     8 */

        /* XXX 4 bytes hole, try to pack */

        union {
                struct bgp_mplsvpn_label_nh blnc;        /*   120    24 */
                struct bgp_mplsvpn_nh_label_bind bmnc;   /*   120    24 */
        } mplsvpn;                                       /*   120    24 */

        /* size: 144, cachelines: 3, members: 19 */
        /* sum members: 137, holes: 2, sum holes: 7 */
        /* last cacheline: 16 bytes */
};
```
Please use instead
```
uint32_t flags;
#define BGP_PATH_XXX
```

Reply:
ok. changes done.

Reply:
OK